### PR TITLE
Create docs for Gradle Worker API

### PR DESCRIPTION
**Pull request by atulgpt**
Fixes https://github.com/detekt/detekt/issues/5843

### CODE REVIEW

File: website/versioned_docs/version-1.23.0/gettingstarted/gradle.mdx [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+    
+    // Specify the base path for file paths in the formatted reports. 
+    // If not set, all file paths reported will be absolute file path.
+    basePath = projectDir
+}
+```
+
+### <a name="gradle-properties">Options for detekt Gradle properties</a>
+
+```groovy
```
Comment (atulgpt) [inline website/versioned_docs/version-1.23.0/gettingstarted/gradle.mdx:300]: Added the doc here

**Comment by cortinico** [comment]
We don't need the whole 1.23 docs (those will be cut once we do the stable),
but just the extra page

**Comment by cortinico** [comment]
You don't need to create anything for version 1.23.0. Just add your content here: https://github.com/detekt/detekt/blob/main/website/docs/gettingstarted/gradle.mdx

The docs for 1.23 will appear as soon as we cut the release version

**Comment by BraisGabin** [comment]
Update them inside `website/docs/gettingstarted/gradle.mdx`. This way, as soon as we cut a new version, the content inside there will be frozen on the 1.23 version.

**Comment by atulgpt** [comment]
> Update them inside `website/docs/gettingstarted/gradle.mdx`. This way, as soon as we cut a new version, the content inside there will be frozen on the 1.23 version.

@BraisGabin / @cortinico done 😅. Thanks for the help

We can optionally also rename the folder `versioned_docs` to `generated_versioned_docs` 🙂

**Comment by cortinico** [comment]
> We can optionally also rename the folder `versioned_docs` to `generated_versioned_docs` 🙂

Nope those are not generated. It's a "copy" of the docs as they are once a given version is released. It still allows us to edit versioned docs if we find a problem in a previous version

**Review by BraisGabin — APPROVED** [review]

**Review by cortinico — APPROVED** [review]

**Pull request merged**